src/applications/search/containers/SearchApp.jsx

```diff
@@ -54,16 +46,24 @@ class SearchApp extends React.Component {
       currentResultsQuery: userInputFromURL,
       page: pageFromURL,
       typeaheadUsed,
+      formWasSubmitted: false,
+      landedWithSearchInput: false,
```

Comment:
Latest changes: added a tracker to flag when we are landing with a search value already filled in (i.e. the global search in the header has been used, and we arrive on the search page with the term filled in).